### PR TITLE
Empty lines should not affect space removal

### PR DESCRIPTION
**Pull request by OrKoN**
For example, previously some commands were formatted with extra whitespace in the CDL file:

```
  ScriptCallFunctionCommand = {
    method: "script.callFunction",
    params: ScriptCallFunctionParameters
  }
```

With this change, it becomes:

```
ScriptCallFunctionCommand = {
  method: "script.callFunction",
  params: ScriptCallFunctionParameters
}
```

I am not sure where those lines containing only spaces come from though.

### CODE REVIEW

**Comment by whimboo** [comment]
It's because of the indendation of the `pre` node as you can see here:
https://github.com/w3c/webdriver-bidi/blob/6f755a2d2c9109871656acfb03864e0405eb63ab/index.bs#L4190-L4194

I've spotted that as well when I refactored the CDDL generation and wanted to do that in a follow-up PR. But thanks that you already took care of it!

I've approved the test jobs and when inspecting the CDDL artifacts it looks all fine and is way better to read!

I'll leave it for @jgraham for the final review.

**Comment by whimboo** [comment]
I wonder if we should also still fix the indentation in the spec file?

**Review by jgraham — APPROVED** [review]
It's probably a reasonable idea to fix the spec, but I don't think we need to block on that.

**Comment by whimboo** [comment]
> It's probably a reasonable idea to fix the spec, but I don't think we need to block on that.

I can take care of in my next PR for CDDL. Just triggered the tests and if all passes I'll go ahead and merge.

**Pull request merged**
